src/cart/src/cart/crt_init.c

```diff
@@ -130,6 +131,13 @@ static int data_init(int server, crt_init_options_t *opt)
 	crt_gdata.cg_na_plugin = CRT_NA_OFI_SOCKETS;
 	crt_gdata.cg_share_na = false;
 
+	srand(d_timeus_secdiff(0) + getpid());
+	start_rpcid = (rand() % 65535) << 16;
```

Comment:
Well, we can do something like 80K RPCs /s per process, so we will overflow it pretty quickly. Why not a 64-bit ID? would consume too much space?

Reply:
changed